### ISSUE TITLE
🛠️ Repo: checkov write-all permissions complaint on .github/workflows/stale.yml (via trunk)

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [eslint/eslint#19356]
## Overview

Discovered in #19355: upon changing the [`.github/workflows/stale.yml` file](https://github.com/eslint/eslint/blob/17f2aaec16d5afbb0d219bce6ae01d7b15d74828/.github/workflows/stale.yml), `trunk check` is newly picking it up and reporting:

```plaintext
.github/workflows/stale.yml:15:1
 15:1  high  Ensure top-level permissions are not set to write-all  checkov/CKV2_GHA_1

Checked 1224 modified files
5 existing issues (use --show-existing to see them)
✖ 1 new security issue
```

I'm not familiar myself with checkov and the docs on https://www.checkov.io/5.Policy%20Index/github_actions.html aren't super informative. But you can reproduce this locally with:

```shell
npx trunk check .github/workflows/stale.yml --show-existing
```

I'd have though the complaint is about this:

https://github.com/eslint/eslint/blob/f47847c1b45ef1ac5f05f3a37f5f8c46b860c57f/.github/workflows/stale.yml#L16-L18

...but the report still happens if the lines are deleted. So I don't know what to do about this.

**Comment by fasttime** [comment]
Have you tried to change the permission for the workflow? That would be adding `permissions: read-all` at top level additionally to the specific permissions for the one job like here:

https://github.com/eslint/eslint/blob/4f08332ac03e51002f8de6da5c5a362608205437/.github/workflows/annotate_pr.yaml#L7-L17

**Comment by nzakas** [comment]
@JoshuaKGoldberg I'd say open a bug with Checkov about this.

**Comment by JoshuaKGoldberg** [comment]
> `permissions: read-all`

👍 That does fix the linting issue, nice! Sent: #19374

> open a bug with Checkov about this

👍 Sent a docs bug: https://github.com/bridgecrewio/checkov/issues/6977

For now I'd propose we mark this as blocked pending Checkov adding those docs.

**Comment by nzakas** [comment]
If we've fixed the problem then I'd say let's close the issue. I don't think we need to track whether or not Checkov updates their docs. Unless we're waiting on something else besides docs from them?

**Comment by fasttime** [comment]
Closing as completed per #19374.
